server/src/test/java/io/crate/planner/DeletePlannerTest.java

```diff
@@ -108,4 +108,12 @@ public void test_delete_where_id_and_seq_missing_primary_term() throws Exception
             VersioninigValidationException.SEQ_NO_AND_PRIMARY_TERM_USAGE_MSG
         );
     }
+
+    @Test
+    public void test_delete_with_subquery_has_delete_type_plan() {
+        MultiPhasePlan plan = e.plan(
+            "delete from users where id in (" +
+                "   select unnest([1, 2, 3, 4]))");
+        assertThat(plan.type(), is(Plan.StatementType.DELETE));
```

Comment:
Isn't the new `test_classify_multiphase_delete_statement` tests the same?

Reply:
Basically yes, intention was to have it covered if Delete/Update planners gets updated to something like MultiPhaseType2 with same bug but it's not really likely to happen. Removing Delete/Update planner tests.